### DISCUSSION TITLE
Welcome to Projekt-Bierzaehler Discussions!

### DISCUSSION THREAD

**Discussion by SebastianZug** [Ifi-Softwareentwicklung-SoSe2022/Projekt-Bierzaehler#7] [General]
Hallo zusammen,
konnten Sie das Problem mit den RFID Chips lösen? Soll ich mich noch darum kümmern?
Sebastian Zug

**Comment by Nagelow** [comment]
Nein, der Versand hätte bis zur Abgabe des Projekts zu lange gedauert. Wenn es ihnen möglich ist bis zum 27. Juli die Chips und Reader zu besorgen, würden wir das dankend annehmen.
